### PR TITLE
fix violations of react/jsx-no-bind in code-studio pd forms

### PR DESCRIPTION
**Pull request by Hamms**

### CODE REVIEW

File: apps/src/code-studio/pd/form_components/FormController.jsx [hunk 1]
```diff
@@ -284,7 +289,7 @@ export default class FormController extends React.Component {
         style={styles.pageButtons}
         items={this.getPageComponents().length}
         activePage={this.state.currentPage + 1}
-        onSelect={i => this.setPage(i - 1)}
+        onSelect={i => this.setPage(i - 1)} // eslint-disable-line react/jsx-no-bind
```
Comment (aoby) [inline apps/src/code-studio/pd/form_components/FormController.jsx:292]: FYI you can get around this one too by setting a data- field to your index, and inspecting that in the callback. For example, set [here](https://github.com/code-dot-org/code-dot-org/blob/a2adb2db73b4a17d6cbc5d4cfd9145eae13ba8e1/apps/src/code-studio/pd/workshop_dashboard/workshop.jsx#L383-L390), inspected [here](https://github.com/code-dot-org/code-dot-org/blob/a2adb2db73b4a17d6cbc5d4cfd9145eae13ba8e1/apps/src/code-studio/pd/workshop_dashboard/workshop.jsx#L199-L199)

Comment (Hamms) [reply apps/src/code-studio/pd/form_components/FormController.jsx:292]: ooooh, I like it

Comment (Hamms) [reply apps/src/code-studio/pd/form_components/FormController.jsx:292]: unfortunately, it's not gonna work here; that `i` is not provided by the component instantiating the Pagination component, but rather by the component itself. What I'd really have to do is something like define a `handleSelectPage(i) { this.setPage(i-1) }` method on FormController and pass that to Pagination and that just seems like a silly thing to do for a rule we're not actually even enforcing

Comment (aoby) [reply apps/src/code-studio/pd/form_components/FormController.jsx:292]: Oh yeah, I see. I misread it. 👍

**Pull request merged**